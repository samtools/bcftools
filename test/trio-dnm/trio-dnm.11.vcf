##fileformat=VCFv4.2
##reference=file:///lustre/scratch118/humgen/resources/ref/Homo_sapiens/GRCh38_15/Homo_sapiens.GRCh38_15.fa
##contig=<ID=chr1,length=248956422>
##contig=<ID=chrX,length=156040895>
##FORMAT=<ID=PL,Number=G,Type=Integer,Description="List of Phred-scaled genotype likelihoods">
##FORMAT=<ID=AD,Number=R,Type=Integer,Description="Allelic depths (high-quality bases)">
##FORMAT=<ID=QS,Number=R,Type=Integer,Description="Phred-score allele quality sum used by `call -mG` and `+trio-dnm`">
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	proband	father	mother
chr1	193463	.	G	A	.	.	.	PL:AD:QS	234,0,84:2,7:120,420	0,9,92:3,0:111,0	0,15,129:5,0:185,0
chr1	10000057	.	C	T	.	.	.	PL:AD:QS	90,0,255:30,30:1116,1116	255,78,0:0,26:0,921	255,78,0:0,26:0,921
chr1	10000057	.	C	T	.	.	.	PL:AD:QS	255,90,0:0,30:0,1116	255,78,0:0,26:0,921	0,114,255:38,0:1364,0
chr1	10697377	.	C	T	.	.	.	PL:AD:QS	255,0,173:4,10:240,566	255,184,0:0,61:0,3553	255,208,0:0,69:0,409
chr1	141907033	.	G	A	.	.	.	PL:AD:QS	54,0,255:53,8:2673,257	0,238,255:79,0:3268,0	0,188,255:128,7:5087,438
chrX	10000804	.	C	G	.	.	.	PL:AD:QS	255,0,85:5,19:174,671	202,0,157:7,9:255,323	0,30,227:10,0:370,0
chrX	53198529	.	A	G,C	.	.	.	PL:AD:QS	255,0,255,255,255,255:28,22,0:1282,963,0	0,148,255,148,255,255:49,0,0:2342,0,0	0,208,255,192,255,255:69,0,1:2728,0,11
chrX	101841247	.	G	A	.	.	.	PL:AD:QS	215,12,0:27,36:108,712	0,93,121:31,0:310,0	0,95,169:69,9:828,90
chrX	141907033	.	G	A	.	.	.	PL:AD:QS	54,0,255:53,8:2673,257	0,238,255:79,0:3268,0	0,188,255:128,7:5087,238
