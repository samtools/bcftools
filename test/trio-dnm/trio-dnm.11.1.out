chr1:193463  DNM=0 . . 	AD=2,7 3,0 5,0 	QS=120,420 111,0 185,0 	VAF=78 0 0 	VA=1 . . 
chr1:10000057  DNM=0 . . 	AD=30,30 0,26 0,26 	QS=1116,1116 0,921 0,921 	VAF=50 0 0 	VA=0 . . 
chr1:10000057  DNM=-0.0953102 . . 	AD=0,30 0,26 38,0 	QS=0,1116 0,921 1364,0 	VAF=100 100 0 	VA=1 . . 
chr1:10697377  DNM=-5.01188e-10 . . 	AD=4,10 0,61 0,69 	QS=240,566 0,3553 0,409 	VAF=29 0 0 	VA=0 . . 
chr1:141907033  DNM=-inf . . 	AD=53,8 79,0 128,7 	QS=2673,257 3268,0 5087,438 	VAF=13 0 5 	VA=1 . . 
chrX:10000804  DNM=-9.99997e-10 . . 	AD=5,19 7,9 10,0 	QS=174,671 255,323 370,0 	VAF=79 56 0 	VA=1 . . 
chrX:53198529  DNM=0 . . 	AD=28,22,0 49,0,0 69,0,1 	QS=1282,963,0 2342,0,0 2728,0,11 	VAF=44 0 0 	VA=1 . . 
chrX:101841247  DNM=-20.5271 . . 	AD=27,36 31,0 69,9 	QS=108,712 310,0 828,90 	VAF=57 0 12 	VA=1 . . 
chrX:141907033  DNM=-5.98923 . . 	AD=53,8 79,0 128,7 	QS=2673,257 3268,0 5087,238 	VAF=13 0 5 	VA=1 . . 
